builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,189 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+BuildRequires:  rust >= 1.47.0
+Requires(pre):  shadow-utils
+Requires:       openssl
+Requires:       moby-engine
+Requires:       moby-cli
+
+#Source0:       https://github.com/Azure/iotedge/archive/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+
+%description
+Azure IoT Edge Module Runtime
+Azure IoT Edge is a fully managed service that delivers cloud intelligence
+locally by deploying and running artificial intelligence (AI), Azure services,
+and custom logic directly on cross-platform IoT devices. Run your IoT solution
+securely and at scale—whether in the cloud or offline.
+
+This package contains the IoT Edge daemon and CLI tool.
+
+%prep
+%setup -q
+
+%build
+IOTEDGE_HOST=unix:///var/lib/iotedge/mgmt.sock
+export IOTEDGE_HOST
```

Comment:
Did we ever find out why these are needed for EFLOW? 
I know without these, IoTEdge will have a hard time connecting to docker... but why?